IndexProactiveControls.md

```diff
@@ -0,0 +1,166 @@
+# Table of contents
+
+<!-- Table of contents generated with markdown-toc https://ecotrust-canada.github.io/markdown-toc/ -->
+
+- [Objective](#objective)
+- [Not adressed CS](#not-adressed-cs)
+- [1. Define Security Requirements](#1-define-security-requirements)
+- [2. Leverage Security Frameworks and Libraries](#2-leverage-security-frameworks-and-libraries)
+- [3. Secure Database Access](#3-secure-database-access)
+- [4. Encode and Escape Data](#4-encode-and-escape-data)
+- [5. Validate All Inputs](#5-validate-all-inputs)
+- [6. Implement Digital Identity](#6-implement-digital-identity)
+- [7. Enforce Access Controls](#7-enforce-access-controls)
+- [8. Protect Data Everywhere](#8-protect-data-everywhere)
+- [9. Implement Security Logging and Monitoring](#9-implement-security-logging-and-monitoring)
+- [10. Handle All Errors and Exceptions](#10-handle-all-errors-and-exceptions)
+
+# Objective
+
+This index have for objective to indicate to help an [OWASP Proactive Controls](https://www.owasp.org/index.php/OWASP_Proactive_Controls) user to clearly identify which cheat sheets are useful for each section during his usage of the OWASP Proactive Controls.
+
+# Not adressed CS
+
+[AJAX Security Cheat Sheet](cheatsheets/AJAX_Security_Cheat_Sheet.md)
+
+[Cross-Site_Request_Forgery_Prevention_Cheat_Sheet](cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md) - Enforce Access Controls? 
+
+[Docker Security Cheat Sheet](cheatsheets/Docker_Security_Cheat_Sheet.md) -  Leverage Security Frameworks and Libraries?
```

Comment:
With Docker, application is often considered as a "docker image" so it can be interesting to reference the docker CS in adequate section in order to give hint to dev team when packaging the app as "container", what do you think guys ?

Reply:
I wasn't able to actually map it to any of the mentioned proactive controls.

Reply:
Docker is probably out of scope of this, the best place to put it is "Leverage Security Frameworks and Libraries" but docker is not a security tool.